src/lib.rs

```diff
@@ -2883,7 +2883,7 @@ impl Build {
     /// Get the archiver that's in use for this configuration.
     ///
     /// This will return a result instead of panicing;
-    /// see [`get_archiver()`] for the complete description.
+    /// see [Self::get_archiver] for the complete description.
```

Comment:
I thought the infra link needs to be quoted by `

Reply:
That is not necessary, it's just for formatting.

Reply:
thx, though I prefer it to be quoted